### DISCUSSION TITLE
Carousel description per image

### DISCUSSION THREAD

**Discussion by arahmitz** [nunocoracao/blowfish#1237] [Q&A]
Hey,

I've been reading the documentation and I wonder - is there a way, to add text per image? Not sure if that's an alt text, as I want it to render like image's bracket content.

I am not sure about how would the syntax looks like, so I imagine something like this:
```md
{{< carousel images="{gallery/03.jpg "This is a photo #3", gallery/01.jpg "This is a photo #1", gallery/02.jpg "This is a photo #2"}  >}}
```

I am somewhat new to webdev, so pardon me if I botchered the syntax.

**Comment by nicholas-alonzo** [comment]
I would also like to know if this is possible, it'd be a great feature
